src/track/macro.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <QtCore>
+#include <memory>
+
+#include "engine/engine.h"
+#include "proto/macro.pb.h"
+#include "track/macroaction.h"
+#include "util/db/dbid.h"
+namespace proto = mixxx::track::io;
+
+const QLoggingCategory macroLoggingCategory("macros");
```

Comment:
- Wrong naming of constants
- Split declaration and initialization, requires to use `extern` in the header. Otherwise the compiler will actually create multiple, unrelated constants, one for each inclusion of the header file.
- Does this constant need to be public?